libs/ledger-live-common/src/transaction/common.ts

```diff
@@ -1,9 +1,19 @@
-import { BigNumber } from "bignumber.js";
+import { deserializeError, serializeError } from "@ledgerhq/errors";
 import type {
+  Account,
   TransactionCommon,
   TransactionCommonRaw,
-} from "../types/transaction";
-import type { Transaction, TransactionRaw } from "../types";
+} from "@ledgerhq/types-live";
+import { BigNumber } from "bignumber.js";
+import { mapValues } from "lodash";
+import { getAccountUnit } from "../account";
+import { formatCurrencyUnit } from "../currencies";
+import type {
+  Transaction,
+  TransactionRaw,
+  TransactionStatus,
+  TransactionStatusRaw,
+} from "../types";
```

Comment:
I don't see how these are imported from local types, since you removed `src/types/transaction.ts` :thinking: 
They should be imported from the types lib anyway, right?

Reply:
TODO: Replace by common types instead

Reply:
Done